torch/testing/_internal/common_utils.py

```diff
@@ -92,6 +92,14 @@ def enable_profiling_mode():
         torch._C._jit_set_profiling_executor(old_prof_exec_state)
         torch._C._jit_set_profiling_mode(old_prof_mode_state)
 
+@contextmanager
+def set_num_profiled_runs(num_runs):
```

Comment:
Nit: for the context manager a name without `set_` might be more appropriate.

Reply:
i guess it does sound better. I'll make a change